zeppelin-web/src/app/tabledata/columnselector_settings.html

```diff
@@ -47,11 +47,10 @@
               style="font-weight: 300; font-size: 13px; margin-left: 1px;"
               ng-style="prop.name === 'xAxis' ? { 'background-color': '#906ebd' } : prop.name === 'yAxis' ? { 'background-color': '#cd5c5c' } : prop.name === 'group' ? { 'background-color': '#5782bd' } : ''">
         {{prop.name}}
-        <a tabindex="0" class="fa fa-info-circle" role="button" popover-placement="top"
+        <a tabindex="0" class="fa fa-info-circle" role="button"
            ng-if="prop.tooltip"
            style="color: white;"
-           popover-trigger="focus"
-           popover-html-unsafe="{{prop.tooltip}}"></a>
+           uib-tooltip="{{prop.tooltip}}"></a>
```

Comment:
I think we need to use `uib-tooltip-html` 

- https://angular-ui.github.io/bootstrap/#!#tooltip

![image](https://user-images.githubusercontent.com/4968473/27505652-4a72d3a2-58e0-11e7-9103-30f8cc61915a.png)